packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4186,18 +4202,28 @@ class InheritedElement extends ProxyElement {
   /// Calls [Element.didChangeDependencies] of all dependent elements, if
   /// [InheritedWidget.updateShouldNotify] returns true.
   ///
-  /// Notifies all dependent elements that this inherited widget has changed.
+  /// Called by [update], immediately prior to [build].
   ///
-  /// [InheritedElement] calls this function if the [widget]'s
-  /// [InheritedWidget.updateShouldNotify] returns true.
+  /// Calls [notifyClients] to actually trigger the notifications.
```

Comment:
Now it calls `updated` which calls `notifyClients` by default.

Reply:
This _is_ updated.